docs/shared-content/scripts/deploy-release-with-prompted-variables-scripts.include.md

```diff
@@ -385,7 +385,7 @@ func main() {
 
 		// check to see if the element display name matches
 		if (element["Control"].(map[string]interface{}))["Name"] == parsedPromptedVariable[0] {
```

Comment:
Would this IF block need to be updated like the variable assignment below?

Reply:
No, here we compare actual variable name, not UUID

Reply:
Ah!  Yes!  You're correct, thank you clarifying.